content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,381 @@
+  - defended by an _innocence_ proof, submitted by the _offending_ validator within a proof submission window measured in blocks
+  - promoted to _fault_ by protocol on individual block finalisation if not refuted by _innocence_ and the _offending_ validator does not already have a _fault_ with a higher or equivalent severity. 
+
+Slashing penalties are computed by protocol and  applied for proven faults at epoch end. The penalty amount is computed based on a base slashing rate and slashing factors including the total number of slashable offences committed in the epoch and the individual _offending_ validator's own slashing history.
+
+Slashing is applied as part of the state finalisation function:
+
+- As each block is finalised, AFD will: promote within protocol constraints new _accusations_ to proven _faults_ after expiry of an _innocence_ proof submission window
+- As the last block of an epoch is finalised, AFD will: apply accountability for _faults_ to _offending_ validators, slashing [self-bonded](/glossary/#self-bonded) and [delegated](/glossary/#delegated) stake  according to Autonity's [Penalty-Absorbing Stake (PAS)](/glossary/#penalty-absorbing-stake-pas) model.
+
+Rewards are paid for reporting a slashable _fault_ at epoch end. The _offending_ validator's share of the epoch's staking rewards is forefeited and paid to the  _reporting_ validator `treasury` account for distribution along with the staking rewards at epoch end.
+
+### Accountability prerequisites 
+
+To participate in AFD a [validator](/glossary/#validator) must be a [consensus committee](/glossary/#consensus-committee) member.
+
+## Accountability and Fault Detection protocol
+
+AFD roles, core concepts, and the lifecycle of accountability event processing from accusation to slashing.
+
+### Validator roles
+
+As consensus committee members, validators play the following roles in AFD processing:
+
+- _reporting_: as the validator reporting a suspected rule infraction and submitting new _accusation_ proofs on-chain
+- _offending_: as the validator committing a suspected rule infraction and submitting new _innocence_ proofs on-chain
+- _committee member_: as a validator in the consensus committee executing Autonity's consensus protocol and for AFD handling and processing accountability events, maintaining system state for accountability events, and computing and applying slashing penalties.
+
+Validators play the following economic roles in AFD processing:
+
+- _loss_: loss of stake, validator reputation, staking reward revenue as the _offending_ validator of a slashed _fault_
+- _beneficiary_: gain of slashing reward as the _reporting_ validator of a slashed _fault_.
+
+Autonity community play the following economic role in AFD processing:
+
+- _beneficiary_: gain of slashed stake token for community funds.
+
+### Protocol primitives
+
+Essential primitives of AFD are accusations, proof of innocence, fault promotion, slashing and severity, and jailing.
```

Comment:
Fault proofs too?

Reply:
That's covered under the entry for **Faults** but this is an error here: 'fault promotion' should be 'faults'. Good catch.

Reply:
Corrected in commit b694214f6ff29b8c9e35d601b45c48eac51cd3ed

Reply:
Can this be resolved @yazzaoui or is further editing required?

Reply:
All good !